packages/features/eventtypes/lib/getPublicEvent.ts

```diff
@@ -37,7 +38,24 @@ const publicEventSelect = Prisma.validator<Prisma.EventTypeSelect>()({
   currency: true,
   seatsPerTimeSlot: true,
   bookingFields: true,
-  team: true,
+  team: {
+    select: {
+      parentId: true,
+      metadata: true,
+      brandColor: true,
+      darkBrandColor: true,
+      slug: true,
+      name: true,
+      logo: true,
+      theme: true,
+      parent: {
+        select: {
+          slug: true,
+          name: true,
+        },
+      },
+    },
+  },
```

Comment:
Before this, it was selecting all of these and more fields, this just specifies the ones needed adding parent